.buildkite/scripts/hardware_ci/run-xpu-test.sh

```diff
@@ -31,6 +31,7 @@ docker run \
     set -e
     echo $ZE_AFFINITY_MASK
     VLLM_USE_V1=1 python3 examples/offline_inference/basic/generate.py --model facebook/opt-125m --block-size 64 --enforce-eager
+    VLLM_USE_V1=1 python3 examples/offline_inference/basic/generate.py --model facebook/opt-125m --block-size 64 --enforce-eager -O3
```

Comment:
What is --enforce-eager -O3 ?

Can we do -O3 with use_cudagraph=False ? (or whatever the new way to disable cudagraphs is?)

Reply:
`--enforce-eager -O3 ` this will use piecewise cuda graph compiler backend, but will not capture cuda graph on cuda device.(I am not certain about current cuda behavior, but it does work on cuda 2 or 3 month ago)
I think in vllm `--enforce-eager` equals to `not use cuda graph`, we don't expose `use_cudagraph` as an vllm arg.

Reply:
make sense! please take a look again. thanks!